Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -65,6 +65,7 @@ type Props = NavigationSceneRendererProps & {
   renderRightComponent: NavigationSceneRenderer,
   renderTitleComponent: NavigationSceneRenderer,
   style?: any;
+  subviewStyle: any;
```

Comment:
I am not an expert in flow, `style` just above is equally optional and typed as `any` so I just followed it.